README.md

```diff
@@ -7,8 +7,7 @@
   </p>
 </h1>
 
-A code search engine for the AI age. SeaGOAT is a local search tool that
-leverages vector embeddings to enable to search your codebase semantically.
+SeaGOAT is a code search engine for the AI age. It is a local search tool that leverages vector embeddings to enable to search your codebase semantically.
```

Comment:
I like this version less than the current one. Though the grammar is simpler, so maybe that's in improvement. But the line is too long now

Reply:
😟